EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1230,273 +1230,273 @@ QUEST_LV_0300_20160310_001229	Even if you can't trust me you should trust Vaidot
-QUEST_LV_0300_20160310_001274	Now, the main culprit is the priest of the religious body of the truth, Dominicas..{nl}Everything will be over when we drive him out.{nl}
-QUEST_LV_0300_20160310_001275	Please lend me your power little more.{nl}Even if we have ways to get him, the villagers are all seduced and due to long torture, we can't use our power..
-QUEST_LV_0300_20160310_001276	He seems to have some power to control the plants freely.{nl}Be careful since he is not to be underestimated.
-QUEST_LV_0300_20160310_001277	Dominicas.. Ran away��{nl}
-QUEST_LV_0300_20160310_001278	But, it's okay.{nl}It's a big harvest that we drove out the religious body of the truth from Elgos Monastery.{nl}
+QUEST_LV_0300_20160310_001233	I am talking about the scripture from the Tree of Truth sect left by Dominikas.{nl}It turns out that impious stuff is still remaining in the monastery.{nl}
+QUEST_LV_0300_20160310_001234	If you see the fake scripture, could you bring it to me?{nl}We are short-handed to deal with it individually...{nl}
+QUEST_LV_0300_20160310_001235	Those at the Tree of Truth sect... polluted the power of the goddess...{nl}This is just blasphemy to the goddess!
+QUEST_LV_0300_20160310_001236	So you've found the fake scripture. Thank you.{nl}Give it to me and I will take care of it.
+QUEST_LV_0300_20160310_001237	S...Stop!{nl}I will do whatever you ask me, so please stop!
+QUEST_LV_0300_20160310_001238	Should I just join the Tree of Truth sect?{nl}If so, would I be able to be relieved from this hunger?
+QUEST_LV_0300_20160310_001239	Lies...{nl}If not... How... The black wall...
+QUEST_LV_0300_20160310_001240	It seems he is suffering undernourishment for a long time.{nl}I can't talk to him now.
+QUEST_LV_0300_20160310_001241	Thank you so much...{nl}I am so sorry for misunderstanding you. I thought something was going wrong...{nl}
+QUEST_LV_0300_20160310_001242	After I gained my consciousness, I felt this mysterious energy from you.{nl}What happened?{nl}
+QUEST_LV_0300_20160310_001243	Ah... So you are the Revelator.{nl}I guess this is why the black wall couldn't block you.
+QUEST_LV_0300_20160310_001244	Thank you. I was able to survive because of you.{nl}Dominikas... He is more cruel than the demons.
+QUEST_LV_0300_20160310_001245	It hasn't been a long time since Dominikas visited Elgos Monastery.{nl}I thought he was just an accompanying monk.{nl}
+QUEST_LV_0300_20160310_001246	Not so long after he came here, the tree vines and the {nl}demons suddenly swept the monastery annex.{nl}The villagers fled there... But, he stopped them appropriately.{nl}
+QUEST_LV_0300_20160310_001247	When I think about it, it was all his self-fabricated scenario.{nl}The raid of the tree vines and the sudden attacks of the demons...{nl}
+QUEST_LV_0300_20160310_001248	As the villagers started to follow him, he quickly showed his instinct.{nl}He started to ask us to create an evil idol and believe in the Divine Tree.{nl}
```

Comment:
"he quickly showed his instinct" -> "he quickly showed his true nature."

"He started to ask us" -> "He asked us"